modules/centrifuge/meta.yml

```diff
@@ -0,0 +1,46 @@
+name: centrifuge
+description: Classifies metagenomic sequence data
+keywords:
+  - classify
+  - metagenomics
+  - fastq
+  - db
+tools:
+  - centrifuge:
+      description: Centrifuge is a classifier for metagenomic sequences.
+      homepage: https://ccb.jhu.edu/software/centrifuge/
+      documentation: https://github.com/DaehwanKimLab/centrifuge
+      doi: 10.1101/gr.210641.116
+      licence: ['GPL v3']
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - db:
+      type: directory
+      description: Centrifuge database
```

Comment:
Given you strip off a bunch of hard-coded suffixes, and requires `tar.gz` files (it seems) this should at least be documented here.

Reply:
Thank you for the suggestions, I am not quite sure what you would expect here, something like:

```
- db:
      type: directory
      description: Centrifuge database in tar.gz format
```

or:

```
- db:
      type: directory
      description: Centrifuge database
      pattern: "*.{tar.gz}"
```

Reply:
Both would be good :D